Database/Corrections/sodQuestFixes.lua

```diff
@@ -140,6 +140,11 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410111,
             [questKeys.zoneOrSort] = sortKeys.HUNTER,
         },
+        [78121] = {
```

Comment:
What this quest also needs is the `preQuestSingle` field, which hides this quest until the quest in `preQuestSingle` is turned in. That field is a table of questIds and if one of the questIds is already complete, then the quest will show.

There is also `preQuestGroup` which requires ALL quests of that table to be complete in order to show the quest. But if there is just a single pre quest, we are using `preQuestSingle`.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5332/commits/6e704ae1a13649bfa253ed3bfb3358d0fe7744ad